src/analysis/souffle/authorization_logic.dl

```diff
@@ -52,7 +56,8 @@ isPrincipal(speaker) :- saysDowngrades(speaker, _, _).
 //-----------------------------------------------------------------------------
 downgrades(path, tag) :- ownsTag(owner, tag), saysDowngrades(owner, path, tag).
 
-isTag(tag) :- ownsTag(_, tag).
+// TODO: Add this back in once ownsTag has a real implementation.
```

Comment:
I think we should get rid of the inference rules for `isTag`. They should only be added as ground facts.

Reply:
This sounds like part of the decision for resolving #126. Basically, do we want to have universe-populating rules with the strict requirement that they only talk about base facts, or do we want to always directly state the contents in our universes? Universe-populating rules are useful when hand-writing tests, but they impose circularity risks. The toil of manually populating our universes disappears when we expect that all such code will be generated. I think this comment indicates that we might converge towards eliminating universe-populating rules entirely.

In any case, are you ok with me deferring this kind of cleanup to when we look at fixing #126 directly?

Reply:
Taking the approval as a yes and resolving this convo.